android/app/src/main/java/app/organicmaps/dialog/EditTextDialogFragment.java

```diff
@@ -128,7 +128,10 @@ public Dialog onCreateDialog(Bundle savedInstanceState)
     // Wait till alert is shown to get mPositiveButton.
     editTextDialog.setOnShowListener((dialog) -> {
       mPositiveButton = editTextDialog.getButton(DialogInterface.BUTTON_POSITIVE);
-      this.validateInput(requireActivity(), mInitialText);
+      final FragmentActivity activity = getActivity();
+      if (activity == null)
```

Comment:
This fix looks like a hack. Is there a more reliable way to fix the issue or prevent it from happening?

Reply:
I have no idea.